packages/xo-web/src/common/xo/migrate-vms-modal/index.js

```diff
@@ -152,8 +151,9 @@ export default class MigrateVmsModalBody extends BaseComponent {
       forEach(vbds, vbd => {
         const vdi = vbd.VDI
         if (!vbd.is_cd_drive && vdi) {
-          mapVdisSrs[vdi] =
-            intraPool && doNotMigrateVdi[vdi] ? this._getObject(vdi).SR : srId
+          mapVdisSrs[vdi] = doNotMigrateVdi[vdi]
```

Comment:
`doNotMigrateVdi` can be empty if  `intraPool` is `false` so we can avoid loop in this case ?